platform-includes/getting-started-verify/javascript.nuxt.mdx

```diff
@@ -5,7 +5,7 @@ Check out the <PlatformLink to="/install/cli-import">`--import` CLI flag</Platfo
 
 To verify that Sentry works on the server-side, add the following snippet on the server-side:
 
-```js {tabTitle:Nitro} {filename:server/sentry-example-api.ts}
+```js {tabTitle:Nitro} {filename:server/api/sentry-example.ts}
```

Comment:
I actually don't have strong opinions towards this change. Everything in the `/server` directory is considered "server-side" and you can add event handlers everywhere. But if it's more common to have an `api` folder, we can update this example and it's fine as well :)

Please make sure to update the snippet further down in the file as well:
```
await useFetch('/sentry-example-api');
```

Reply:
I don't think you can add event handlers everywhere. See this reproduction: https://stackblitz.com/edit/github-hxpfgvjj?file=app.vue

Also, `$fetch` should be used instead of `useFetch` when it is called at a time at which the component is already mounted.

In the default nuxt setup, imports from `#imports` are not necessary due to Nuxt's auto-imports.

Lastly, I've marked the api endpoint so that it only applies to `GET` requests which is considered good practice afaik as it decreases the api surface.